source/Camera/Basic/zividApplication.m

```diff
@@ -0,0 +1,29 @@
+function application = zividApplication(folder)
+
+    % application = zividApplication(folder)
+    %
+    % Create Zivid Application
+    %
+    % INPUT:
+    % folder - path to Zivid\bin folder, default: C:\Program Files\Zivid\bin
+    %
+    % OUTPUT:
+    % application - Zivid Application
+
+    if nargin < 1
+        folder = 'C:\Program Files\Zivid\bin';
+    end
+
+    global ZIVID_APPLICATION
+
+    if isempty(ZIVID_APPLICATION)
+        addpath(folder);
+        NET.addAssembly([folder,'\ZividNET.dll']);
+        NET.addAssembly([folder,'\ZividVis3DNET.dll']);
+        import Zivid.*;
```

Comment:
Might want to add:
import Zivid.NET.CaptureAssistant.*
import Zivid.NET.HandEye.*
as these nested classes does not appear without it?

Reply:
The capture assistant sample worked without having to import Zivid.NET.CaptureAssistant.* Does that mean it is not needed?
I am not sure about the HandEye - let's not add this one before we make a sample for it. Maybe it works without having to add it.

Reply:
Done